termux-app/terminal-term/src/main/java/com/termux/app/TermuxActivity.java

```diff
@@ -106,6 +106,10 @@
 
     TermuxPreferences mSettings;
 
+    // Don't attempt to unbind from the service unless the client has received some
+    // information about the service's state.
+    private boolean shouldUnbind;
```

Comment:
Can we explicitly set this to false?

Reply:
Good catch, we should set it's initial value to false.

Reply:
Primitive booleans in Java are automatically set to false, but I personally think it's better to show that than to expect someone reading it in the future to know that tidbit.